app/client/components/delivery/address/formValidation.tsx

```diff
@@ -30,16 +30,18 @@ export const isFormValid = (
   const postcodeEnteredCheck =
     formData.postcode.length > 0 && formData.postcode.length < 20;
 
-  const withinM25Check =
-    subscriptionsNames.includes("home delivery subscription") &&
-    postcodeEnteredCheck &&
-    !isPostcodeOptional("GB") &&
-    isHomeDeliveryInM25(formData.postcode);
+  const withinM25Check = () => {
+    return !subscriptionsNames.includes("home delivery subscription")
+      ? true
+      : postcodeEnteredCheck &&
+          !isPostcodeOptional("GB") &&
+          isHomeDeliveryInM25(formData.postcode);
```

Comment:
this function name is a touch confusing, if all it's checking is whether the postcode is in M25 the function name shouldn't mention home delivery since that part of the check is happening elsewhere